src/main/java/frc/trigon/robot/utilities/Conversions.java

```diff
@@ -9,7 +9,11 @@ public static double hundredMsToMin(double ms) {
         return hundredMsToSec(ms) * 60;
     }
 
-    public static double hundredMsToTickPerMin(double ms) {
-        return hundredMsToMin(ms) * 2048;
+    public static double ticksToRevolutions(double ticks) {
+        return ticks / 4048;
```

Comment:
2048 not 4048

Reply:
there are all sorts of ticks. This should be maybe falconTicks

Reply:
falcon ticks are 2048. the 4 is a typo I assume. but actually, make a constant at top of file